site/docs/README.md

```diff
@@ -84,5 +83,11 @@ Works! :tada:
 
 ---
 
-grammY supports the Telegram Bot API 6.6 which was [released](https://core.telegram.org/bots/api#march-9-2023) on March 9, 2023.
+<ThankYou />
+
+<div style="font-size: 0.75rem;">
+
+© 2021-{{ new Date().getFullYear() }} &middot; grammY supports the Telegram Bot API 6.6 which was [released](https://core.telegram.org/bots/api#march-9-2023) on March 9, 2023.
```

Comment:
I'm not sure if we can simply update the copyright automatically lol

Reply:
Isn’t that server-rendered?

Reply:
It's both. The entire site is rehydrated client-side. Which is why I like islands so much. Remember the random quote above? It changes per visit.

Reply:
Sad

Reply:
Indeed